keyboards/printedpad/printedpad.c

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 Noah Beidelman (@noahbei)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "printedpad.h"
+
+void board_init(void) {
+  SYSCFG->CFGR1 |= SYSCFG_CFGR1_I2C1_DMA_RMP;
+}
+
+const uint8_t max_layer = 3;
+uint8_t current_display_mode = 0;
+
+bool key_pressed = false;
+
+bool process_record_kb(uint16_t keycode, keyrecord_t *record) {
+    if (!process_record_user(keycode, record)) {
+        return false;
+    }
+#ifdef OLED_ENABLE
+    if (record->event.pressed)
+        key_pressed = true;
+    else
+        key_pressed = false;
```

Comment:
You could just do this: 
```suggestion
    key_pressed = record->event.pressed
```